build.gradle

```diff
@@ -94,7 +95,7 @@ dependencies {
     compile group: 'com.github.hmcts', name: 'java-logging', version: versions.reformLogging
     compile group: 'com.github.hmcts', name: 'cmc-pdf-service-client', version: '7.0.0'
 
-    implementation 'com.github.hmcts:sscs-common:4.6.22'
+    implementation 'com.github.hmcts:sscs-common:4.6.45-RC-SSCS-10166'
```

Comment:
Make sure before this is merged to master this is changed back as I don't think this requires formal testing so don't want it to be missed!